src/commands/info.rs

```diff
@@ -19,14 +18,24 @@ pub struct AuthStatus {
 
 #[derive(Serialize, Default)]
 pub struct ConfigStatus {
```

Comment:
Removed `HashMap` here to preserve reproducible json values order for integration tests.